packages/design-picker/src/components/theme-style-variations-buttons/theme-style-variations-button.tsx

```diff
@@ -0,0 +1,48 @@
+import { createElement, useMemo } from 'react';
+import type { ThemeStyleVariation, ThemeStyleVariationSettingsColorPalette } from '../../types';
+import './style.scss';
+
+interface ThemeStyleVariationsButtonProps {
+	tagName?: string;
+	variation?: ThemeStyleVariation;
+}
+
+const ThemeStyleVariationsButton: React.FC< ThemeStyleVariationsButtonProps > = ( {
```

Comment:
Hmm, technically it is not a "button". What about `ThemeStyleVariationBadge` instead?

Reply:
Makes sense 👍